src/components/Header.tsx

```diff
@@ -62,10 +62,12 @@ export const Header: FC = memo(props => {
       <Menu>
         <MenuItem onClick={handleBookmarksVisibilityClick}>
           {isShowingHiddenBookmarks ? <StyledHideIcon /> : <StyledShowIcon />}
+          <ToolTip >{isShowingHiddenBookmarks ? 'Hide hidden bookmarks' : 'Show hidden bookmarks'}</ToolTip>
         </MenuItem>
         <Separator />
         <MenuItem onClick={handleThemeSwitchClick}>
           <StyledColorLensIcon />
+          <ToolTip >Change theme colors</ToolTip>
```

Comment:
Please correct the opening tag

```suggestion
          <ToolTip>Change theme colors</ToolTip>
```